### DISCUSSION TITLE
What's the state/plan regarding documentation?

### DISCUSSION THREAD

**Discussion by coljac** [Ortho4XP/Ortho4XP#35] [Ideas]
I'm another random user happy to help with this project - though I have expertise in python and linux and so hopefully can focus there. But if new life is being breathed into this project (thank you to those who have done so, I followed that discussion with interest), it might be good to centralise and revise the documentation. As far as I can tell it's a combination of X-Plane forums, youtube, some text files in the repo, etc. I think a proper manual would be a real benefit.

**Comment by artesim-zz** [comment]
Hi !

Yes, I think OldSubSailor brought up the same point.
Maybe the wiki section could be used for that ? Or maybe just markdown files alongside the code, centralized in a /docs folder ?
What do you think ?
If you feel like starting something, you're very welcome !

**Comment by girotobial** [comment]
Yes I think a good manual / getting started guide would be of great benefit. 

We could use the wiki here, or setup something with readthedocs.

**Comment by meznak** [comment]
Either would be a good solution, though I tend to prefer keeping documentation close to the project.

We should also consider what we're documenting. User instructions are certainly useful, but we should also consider style and contributing guides.
